backend/src/main/kotlin/app/ehrenamtskarte/backend/regions/webservice/schema/types/Region.kt

```diff
@@ -5,5 +5,5 @@ data class Region(
     val id: Int,
     val prefix: String,
     val name: String,
-    val regionIdentifier: String
+    val regionIdentifier: String?
```

Comment:
@maxammann @Schedulaar does this have any implications for the frontend/administration? Anywhere this needs to be handled?

Reply:
It could be used in the `administration` frontend. But that would be no problem, because that does not need to keep compatibility.